pkg/api/v1/types.go

```diff
@@ -1649,6 +1649,8 @@ type Endpoints struct {
 type EndpointSubset struct {
 	// IP addresses which offer the related ports.
 	Addresses []EndpointAddress `json:"addresses,omitempty"`
+	// IP addresses which offer the related ports but have not yet passed readiness checks
```

Comment:
Technically they could have stopped being ready (aka died) too, right? It's not just bootstrapping nodes. Can we clarify?